formats/financial_releases_campaign/publisher/details.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "image",
+     "body"
+   ],
+  "properties": {
+     "image": {
+       "type": "object",
+       "additionalProperties": false,
+       "properties": {
+         "url": {
+           "type": "string",
+           "format": "uri"
+         },
+         "alt_text": {
+           "type": "string"
+         }
+       }
+     },
+    "header_legal_disclaimer": {
```

Comment:
it has been added by this [new commit](https://github.com/alphagov/govuk-content-schemas/commit/88f856a81eef711e28cce581c43a2cce8e853b98)